src/main/java/mekanism/common/util/MekanismUtils.java

```diff
@@ -1493,7 +1489,7 @@ public static boolean hasUsableWrench(EntityPlayer player, BlockPos pos)
 				return true;
 			}
 			
-			if(isCoFHHammer(tool.getItem()) && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
+			if(isCoFHHammer(tool.getItem())) // TODO Implement CoFH Hammer && ((IToolHammer)tool.getItem()).isUsable(tool, player, pos))
```

Comment:
It's now not part of the RedstoneFlux API, but from one of the other mods. Letting it stay in the api folder isn't going to work either as it's now part of another mod with probably another name. If support is needed, we need to soft depend on one of the other mods.